stories/molecules/most-active-contributors-card.stories.tsx

```diff
@@ -100,7 +109,7 @@ function generateData() {
 
     return {
       ...user,
-      totalContributions: Object.values(user.contributions).reduce((acc, curr) => acc + curr, 0),
+      total_contributions: Object.values(user.contributions).reduce((acc, curr) => acc + curr, 0),
```

Comment:
Updating Storybook mock data with the right shape of data for the `ContributorStat` type.